packages/lists/src/components/FixedList/FixedList.tsx

```diff
@@ -0,0 +1,164 @@
+ * a callback to add and modify the calculated materialized range, for example in order to always render
+ * a focused item, no matter whether it is currently in view or not.
+ * @param props The FixedList props
+ */
+function getMaterializedItemRanges(props: IFixedListProps): IItemRange[] {
+  const { viewportState, overscanHeight, onGetMaterializedRanges, itemHeight, itemCount } = props;
+
+  const { isScrolling, scrollDirection } = viewportState;
+
+  const visibleRange = getVisibleItemRange(props);
+
+  // Add overscan
+  const overscanCount = Math.ceil(overscanHeight / itemHeight);
+
+  visibleRange.startIndex = Math.max(
+    0,
+    visibleRange.startIndex -
+      (!isScrolling || scrollDirection[Axis.Y] === ScrollDirection.backward ? overscanCount : TRAILING_OVERSCAN_COUNT_WHILE_SCROLLING)
+  );
+  visibleRange.endIndex = Math.min(
+    itemCount - 1,
+    visibleRange.endIndex +
+      (!isScrolling || scrollDirection[Axis.Y] === ScrollDirection.forward ? overscanCount : TRAILING_OVERSCAN_COUNT_WHILE_SCROLLING)
+  );
+
+  let materializedRanges = [visibleRange];
+
+  // Add custom materialized ranges (e.g. for currently focused item that is out of view)
+  if (onGetMaterializedRanges) {
+    materializedRanges = onGetMaterializedRanges(materializedRanges);
+  }
+
+  return materializedRanges;
+}
+
+/**
+ * Calculates the number of materialized items given the array of materialized ranges.
+ * Used to create an array of the correct size when rendering the materialized items.
+ * @param props The FixedList props
+ */
+function getMaterializedItemsCount(materializedRanges: IItemRange[]): number {
+  let materializedItemsCount = 0;
+
+  for (const materializedRange of materializedRanges) {
+    const { startIndex, endIndex } = materializedRange;
+
+    materializedItemsCount += endIndex - startIndex + 1;
+  }
+
+  return materializedItemsCount;
+}
+
+/**
+ * A simple virtualized List component which assumes that all its items have the same height.
+ */
+export const FixedList = React.memo((props: IFixedListProps) => {
+  const { itemCount, itemHeight, onRenderItem } = props;
+
+  const materializedRanges = getMaterializedItemRanges(props);
+  const materializedItemsCount = getMaterializedItemsCount(materializedRanges);
+
+  const children = new Array(materializedItemsCount);
+
+  let childIndex = 0;
+  for (const materializedRange of materializedRanges) {
+    const { startIndex, endIndex } = materializedRange;
+
+    for (let i = startIndex; i <= endIndex; i++) {
+      children[childIndex] = onRenderItem(i, {
+        position: 'absolute',
+        width: '100%',
+        height: `${itemHeight}px`,
+        transform: `translate(0, ${i * itemHeight}px)`
+      });
+
+      childIndex++;
+    }
+  }
+
+  const style: React.CSSProperties = {
+    position: 'relative',
+    height: `${itemCount * itemHeight}px`
+  };
+
+  return <div style={style}>{children}</div>; // tslint:disable-line:jsx-ban-props
```

Comment:
Folks may want to override this tag to not be a `<div/>` but we can handle that in a future iteration.

Reply:
Yes, that's a good suggestion. Let's revisit it for a future iteration.